mesheryctl/pkg/utils/helpers.go

```diff
@@ -450,3 +450,26 @@ func ContentTypeIsHTML(resp *http.Response) bool {
 	}
 	return false
 }
+
+func SearchAndReplace(path, oldString, newString string) error {
```

Comment:
Can use use more descriptive parameters? e.g. 

"currentChannel" and "newChannel"

"path" is the Meshery Configuration file, correct? If so, could we rename to "mesheryConfig" or similar variable name?